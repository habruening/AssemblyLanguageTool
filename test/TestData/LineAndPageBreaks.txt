Line  1, LF
Line  2, CRLine  3, CR+LF
Line  4, CR+CRLine  6, LF+LF

Line  8, LF+CR
Line 10, FFLine 10, FF+LF
Line 11, FF+CRLine 12, FF+CR+LF
Line 13, FF+LF+CR
Line 15, CR+LF+FF+CR+LF

Line 17 