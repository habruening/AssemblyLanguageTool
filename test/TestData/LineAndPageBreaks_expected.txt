1: Line  1, LF
2: Line  2, CR
3: Line  3, CR+LF
4: Line  4, CR+CR
5: 
6: Line  6, LF+LF
7: 
8: Line  8, LF+CR
9: 
10: Line 10, FF
11: Line 10, FF+LF
12: Line 11, FF+CR
13: Line 12, FF+CR+LF
14: Line 13, FF+LF+CR
15: 
16: Line 15, CR+LF+FF+CR+LF
17: 
18: Line 17 